client/state/data-layer/wpcom/me/index.js

```diff
@@ -8,6 +8,7 @@
 import { mergeHandlers } from 'state/action-watchers/utils';
 import account from './account';
 import block from './block';
+import blocks from './blocks';
```

Comment:
If we change the import to `./blocks/sites` instead, we won't need `/me/blocks/index.js` at all.